src/domain/customers/tableTools/customerMessageForm.module.scss

```diff
@@ -0,0 +1,33 @@
+@import 'spacings';
+
+.form {
+  > * {
+    margin: units(1) 0;
+  }
+  width: units(48);
+}
+
+.formActionButtons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.formActionButtonsRight {
+  display: flex;
+  justify-content: flex-end;
+
+  > button {
+    margin-left: units(0.5);
+  }
+}
+
+.template {
+  width: units(16);
+}
+
+.message {
+  textarea {
```

Comment:
I prefer if we can target this `textarea` via a className instead of its tag name

Reply:
We cannot target it with class because of CSS modules (that classname is generated by HDS CSS modules). This is one of the drawbacks with CSS modules :(

Reply:
But the `TextArea` in HDS accepts custom `className`s:

> export declare type TextAreaProps = TextareaHTMLAttributes<HTMLTextAreaElement> & {
    /**
     * Additional class names to apply to the textarea
     */
    className?: string;

Reply:
Tried to do this with class but HDS TextArea className is applied to the wrapper div only which doesn't work for us :(